doc/overview/build.md

```diff
@@ -0,0 +1,127 @@
+# Endless OS 7 build process
+
+This document gives an overview of the tools used to build Endless OS 7.
+
+For guidance on how to build EOS7, see: [`doc/howto/build.md`](doc/howto/build.md).
+
+## OSTree build stage
+
+This stage is defined in
+[eos-build-meta.git](https://github.com/endlessm/eos-build-meta/) using the BuildStream build tool.
+
+It uses some 3rd party BuildStream plugins, which are listed in `project.conf`.
+
+It uses elements from Freedesktop SDK and GNOME Build Metadata via the junction
+elements `freedesktop-sdk.bst` and `gnome-build-meta.bst`.
+
+More details on each of these concepts below.
+
+### BuildStream
+
+BuildStream's documentation is available at:
+<https://docs.buildstream.build/>.
```

Comment:
Which dialect of MarkDown is that? I never encountered it (there are more below).

Reply:
I guess it's Github Flavoured Markdown - https://github.github.com/gfm/#what-is-github-flavored-markdown-. This is an "autolink" But, I advise not reading the spec too much :-)